treeherder/perf/alerts.py

```diff
@@ -102,6 +102,12 @@ def generate_new_alerts_in_series(signature):
                 if t_value == float('inf'):
                     t_value = 1000
 
+                type_name = signature.option_collection.option.name
+                title = signature.suite + " " + signature.test + " "
+                title = title + signature.platform.platform + " "
+                title = title + type_name
```

Comment:
I would use the python `format` argument to construct this:

https://docs.python.org/2/library/stdtypes.html#str.format